src/plugins/intel_gpu/src/graph/include/edgpsi_inst.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+#include "intel_gpu/primitives/edgpsi.hpp"
+#include "primitive_inst.h"
+
+namespace cldnn {
+template <>
+struct typed_program_node<edgpsi> : public typed_program_node_base<edgpsi> {
+    using parent = typed_program_node_base<edgpsi>;
+
+public:
+    using parent::parent;
+
+    program_node& input() const { return get_dependency(0); }
+
+    program_node& im_info() const { return get_dependency(0); }
```

Comment:
there are two referencing functions for the same data. (input/im_info) could you merge them ?